website/addons/s3/templates/s3_node_settings.mako

```diff
@@ -64,20 +73,6 @@
       </div>
       </div>
     </div>
-    <div data-bind="if: showCreateCredentials">
```

Comment:
It looks like this was the only place `showCreateCredentials` was being used. Can you remove it from the `s3NodeConfig` view model?